### DISCUSSION TITLE
Replace aioredis with redis package

### DISCUSSION THREAD

**Discussion by applied-mathematician** [fastapi-users/fastapi-users#1012] [Ideas]
The currently used `aioredis` package had its code incorporated into Redis's Python package `redis` (as of v4.2) to gain the async support.  As a result, `aioredis` will not be well maintained.  The code changes necessary will take only a few minutes since the function names and parameters are the same.

**Comment by frankie567** [comment]
Hello @applied-mathematician! Thank you for the bump on this, I wasn't aware of that! Seems indeed sensible!

**Comment by frankie567** [comment]
Implemented in [v10.0.7](https://github.com/fastapi-users/fastapi-users/releases/tag/v10.0.7)

**Reply by applied-mathematician** [reply]
Thanks!
